utils/auth-utils.ts

```diff
@@ -1,9 +1,20 @@
-import { Session, useSession as nextAuthUseSession } from "next-auth/client";
+import {
+  Session,
+  useSession as nextAuthUseSession,
+  getSession as nextAuthGetSession,
+} from "next-auth/client";
 import { SessionUser } from "&server/models/SessionUser";
+import { NextApiRequest } from "next";
 
 export type NPPSession = Session & { user: SessionUser };
 
+// Client side
```

Comment:
These are the preferred methods for fetching sessions; it will cast them to the correct type